content/docs/replication/deployment/powerflex.md

```diff
@@ -36,15 +36,16 @@ Run the following commands to verify that everything is installed correctly:
   Pods should be `READY` and `RUNNING`
 * Check that the controller config map is properly populated
     ```shell
-    
     kubectl get cm -n dell-replication-controller dell-replication-controller-config -o yaml
     ```
   `data` field should be properly populated with cluster-id of your choosing
   and, if using multi-cluster installation, your `targets:` parameter should be
   populated by a list of target cluster IDs.
 
 
-If you don't have something installed or something is out-of-place, please refer to installation instructions [here](../installation).
+If you don't have something installed or something is out-of-place, please refer
+to installation instructions in [installation-repctl](../install-repctl) or
+[installation](../installation).
```

Comment:
Could you revert this change? We updated the installation part of docs recently,